### PR TITLE
Add default prometheus address

### LINKED ISSUE #1128: Prometheus Graphs component should be configurable via feature toggle
**Description**

The API proxy endpoint for Prometheus Graphs is not configurable at the Feature Toggle level

**Expected result**

We should be able to configure and change the  API proxy endpoint for Prometheus Graphs

**Actual result**

The API proxy endpoint for Prometheus Graphs is not configurable at the Feature Toggle level

**Steps to reproduce**

**Troubleshooting**

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

test it also on `nkyma` where the path is `api/v1/namespaces/kyma-system/services/monitoring-prometheus:http-web/proxy/api/v1`
Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves #1128 #1135

### CODE REVIEW

File: core-ui/src/shared/components/StatsGraph/StatsPanel.js [hunk 1]
```diff
@@ -136,6 +144,10 @@ export function StatsPanel({ type, ...props }) {
     return '';
   }
 
+  let path = features.PROMETHEUS?.config?.path;
```
Comment (qbalukom) [inline core-ui/src/shared/components/StatsGraph/StatsPanel.js:147]: wouldn't it be better to read the path inside the `usePrometheus` hook? It's read from microfrontend context anyway - passing the path through these components seems to only add unnecessary complexity

File: core/src/assets/defaultConfig.json [hunk 2]
```diff
@@ -61,7 +61,10 @@
         }
       },
       "PROMETHEUS": {
-        "isEnabled": true
+        "isEnabled": true,
+        "config": {
+          "path": "api/v1/namespaces/kyma-system/services/monitoring-prometheus:web/proxy/api/v1"
```
Comment (qbalukom) [inline core/src/assets/defaultConfig.json:66]: do we need actually the whole path?
both the prefix (`api/v1/namespaces/kyma-system/services/`) and the suffix (`/proxy/api/v1`) are very unlikely to change - as they're part of the kubernetes and prometheus implementations respectively.
The only two parts that can easily change on a per-cluster basis are service name and port name. I feel that would be enough. Either as a single value (`monitoring-prometheus:web`) or two, one for service name (`monitoring-prometheus`) one for port (`web`)

**Review by qbalukom — CHANGES_REQUESTED** [review]

File: docs/features.md [hunk 3]
```diff
@@ -342,19 +342,38 @@ The **match** keys and **messageSrc** must use the format described in the [`jso
...
+  ```bash
+  "PROMETHEUS": {
+    "isEnabled": true,
+    "config": {
+      "path": "api/v1/namespaces/kyma-system/services/monitoring-prometheus:web/proxy/api/v1"
+    }
+  },
+  ```
+
+- **VISUAL_RESOURCES** – configures if the resource graphs should be rendered at resources details views.
```
Comment (grego952) [inline docs/features.md:373]:
```suggestion
- **VISUAL_RESOURCES** – determines if the resource graphs should be rendered at a resource details view.
```

**Review by grego952 — APPROVED** [review]

**Review by qbalukom — APPROVED** [review]

**Pull request merged**